src/main/scala/io/gatling/sbt/settings/gatling/EnterpriseSettings.scala

```diff
@@ -55,6 +57,7 @@ object EnterpriseSettings {
       config / enterprisePackageId := sys.props.get("gatling.enterprise.packageId").getOrElse(""),
       config / enterpriseTeamId := sys.props.get("gatling.enterprise.teamId").getOrElse(""),
       config / enterpriseSimulationId := sys.props.get("gatling.enterprise.simulationId").getOrElse(""),
+      config / waitForRunEnd := jl.Boolean.getBoolean("gatling.enterprise.waitForRunEnd"),
```

Comment:
```suggestion
      config / waitForRunEnd := sys.props.get("gatling.enterprise.waitForRunEnd").flatMap(_.toBooleanOption).getOrElse(false),
```

What happen when it's not `"true"` or `"false"`? (sample: `"oops"`)

Reply:
In the end, the same thing as with your suggestion: `false` by default.